.github/workflows/release.yml

```
@@ -0,0 +1,105 @@
+        sudo apt-get update
+        sudo apt-get install -y gcc-aarch64-linux-gnu
+
+    - name: Build binary
+      env:
+        GOOS: linux
+        GOARCH: ${{ matrix.arch }}
+        CGO_ENABLED: 1
+      run: |
+        cd src
+        OUTPUT_NAME="digests-api-linux-${{ matrix.arch }}"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)